catroid/build.gradle

```diff
@@ -169,6 +169,7 @@ dependencies {
     compile 'com.koushikdutta.async:androidasync:2.+'
     compile 'com.sothree.slidinguppanel:library:3.3.0'
     compile 'com.squareup.picasso:picasso:2.5.2'
+    compile 'com.rmtheis:tess-two:8.0.0'
```

Comment:
make this a testCompile so its not compiled into the debug and prod versions aswell.